src/backend/replication/walsender.c

```diff
@@ -1902,6 +1907,90 @@ ProcessStandbyReplyMessage(void)
 						applyPtr,
 						replyTime,
 						replyRequested);
+
+	elog(LOG, "ProcessStandbyReplyMessage: writelsn %X/%X",
+					LSN_FORMAT_ARGS(writePtr));
+	elog(LOG, "ProcessStandbyReplyMessage: flushlsn %X/%X",
+					LSN_FORMAT_ARGS(flushPtr));
+	elog(LOG, "ProcessStandbyReplyMessage: applylsn %X/%X",
+					LSN_FORMAT_ARGS(applyPtr));
+}
+
+// This message is a zenith extension of postgres replication protocol
+// TODO unify with ParseZenithFeedbackMessage?
```

Comment:
+1 for reusing ParseZenithFeedbackMessage

Reply:
Done